hazelcast/src/main/java/com/hazelcast/nio/tcp/TcpIpConnectionManager.java

```diff
@@ -78,9 +79,9 @@ public TcpIpConnectionMonitor createNew(Address endpoint) {
 
     private final boolean socketNoDelay;
 
-    private final ConcurrentMap<Address, Connection> connectionsMap = new ConcurrentHashMap<Address, Connection>(100);
+    private final ConcurrentHashMap<Address, Connection> connectionsMap = new ConcurrentHashMap<Address, Connection>(100);
```

Comment:
well spotted! It was a quick hack to Checkstyle stop bitching about "too many dependencies". 
(yes, I'm a terrible person)

Let me find a better solution.